saleor/graphql/account/mutations/account.py

```diff
@@ -36,18 +43,46 @@ class Arguments:
             description="Fields required to create a user.", required=True
         )
 
+    requires_confirmation = graphene.Boolean()
```

Comment:
Let's add some description here as well e.g. "Informs whether users need to confirm their email address."